src/datasource.ts

```diff
@@ -36,7 +36,7 @@ export class AdxDataSource extends DataSourceWithBackend<KustoQuery, AdxDataSour
   private backendSrv: BackendSrv;
   private templateSrv: TemplateSrv;
   private defaultOrFirstDatabase: string;
-  url?: string;
+  private url?: string;
```

Comment:
I think we want to keep this public because we use it here https://github.com/grafana/azure-data-explorer-datasource/blob/master/src/components/ConfigEditor/index.tsx#L43

Reply:
I actually had to refactor that line of code in [this](https://github.com/grafana/azure-data-explorer-datasource/pull/231/files) PR, so it should be fine I think.